pyodide-build/pyodide_build/buildall.py

```diff
@@ -33,7 +33,9 @@ class BasePackage:
     dependencies: List[str]
     unbuilt_dependencies: Set[str]
     dependents: Set[str]
-    unvendored_tests: Optional[bool] = None
+    unvendored_tests: Optional[Path] = None
+    file_name: Optional[str] = None
+    install_dir: str = "site"
```

Comment:
```suggestion
    install_dir: Path = Path("site")
```
?

Reply:
Well this is a choice between "site" ==> `SITE_PACKAGES` and "lib" ==> `STD_LIB`. I look up the proper locations for `SITE_PACKAGES` from `site.getsitepackages()` and for `STD_LIB` from `sysconfig.get_path("stdlib")` at the top of the file. Not sure if using a string for the setting is standard or what.